pywatts/modules/models/profile_neural_network.py

```diff
@@ -78,7 +78,8 @@ def set_params(self, epochs=None, offset=None, batch_size=None, validation_split
 
     def transform(self, **kwargs) -> xr.DataArray:
         """
-        Forecast the electrical load for the given input.
+        Forecast the electrical load for the given input. Note historical_input, trend, and profile has to be provided.
```

Comment:
Please change:
Forecast the electrical load for the given input. Note that historical_input, trend, and profile have to be provided.